docs-compose/overwatch_docs/content/DataEngineer/Productionizing.md

```diff
@@ -55,6 +56,240 @@ created each run but it will be replaced on each subsequent run. If you would li
+| snapshotType    | String | No       | NA                   | Type of Snapshot to be performed. "Full" for Full Snapshot, "Incremental" for Incremental Snapshot                                                 |
+| pipeline        | String | Yes      | "Bronze,Silver,Gold" | Define the Medallion Layers. Argument should be in form of "Bronze, Silver, Gold"(All 3 or any combination of them)                                |
+| cloneLevel      | String | Yes      | "DEEP"               | Clone Level for Snapshot. By Default it is "DEEP". You can also specify "SHALLOW" Clone                                                            |
+| tablesToExclude | String | Yes      | ""                   | Array of table names to exclude from the snapshot. Tables should be separated with ",". This is the table name only - without the database prefix. |
+
+After snapshot process is done the tables from Source ETL Database will be cloned in the target prefix location. The structure for target prefix location would be like below:
+
+![Snapshot_Target](/images/DataEngineer/snapshotTarget.png)
+
+All the tables data would be there in "targetPrefix/data" folder as shown below:
+
+![Snapshot_Target_Data](/images/DataEngineer/snapshotTarget_Data.png)
+
+If you look into the targetPrefix folder there would one another subdirectory called "clone_report". This folder contains the data regarding the report to show the status of your snapshot process for each table:
+
+![clone_report](/images/DataEngineer/clone_report.png)
+
+##### Snapshot through Databricks Job
+
+Snapshot process can also be configured using Databricks Job. For this we need to run SnapshotRunner Class.
+
+Below is the screenshot of the Job through which we run the snapshot process.
+
+1. **Type** - Jar
+2. **Main class** - com.databricks.labs.overwatch.pipeline.SnapshotRunner
+3. **Parameters for the Job** - Same as mentioned above for `Snapshot through Databricks Notebook`
+
+![Snapshot_Job](/images/DataEngineer/snapshot_job.png)
+
+
+#### Validation Functionality
+This is a special module in Snapshot Process. By default, it is part of Snapshot.process() but if user want to run it separately they can run it separately 
+to check whether the configuration they would provide for Snapshot process, are correct or not.
+
+Below is the code snippet for the validation function
+
+```scala
+import com.databricks.labs.overwatch.pipeline.Snapshot
+val sourceETLDB = "ow_bronze_snapshot_etl"
+val pipeline = "Bronze,Silver,Gold"
+val snapshotType = "Incremental"
+Snapshot.isValid(sourceETLDB,snapshotType,pipeline)
+```
+
+Upon running the above function you will get proper output whether the input parameters you would be using for Snapshot process,passed the validation check or not. One example of the same is as below:
```

Comment:
Change to: "This function will return true if the configuration is valid, and will also output relevant info for the fields checked. For example:"

Reply:
Done